src/main/scala/no/ndla/searchapi/service/search/SearchService.scala

```diff
@@ -42,12 +42,12 @@ trait SearchService {
       val articleType = SearchApiProperties.SearchDocuments(SearchType.Articles)
       val draftType = SearchApiProperties.SearchDocuments(SearchType.Drafts)
       val learningPathType = SearchApiProperties.SearchDocuments(SearchType.LearningPaths)
-      hit.`type` match {
-        case `articleType` =>
+      hit.sourceAsMap.get("type") match {
+        case Some(`articleType`) =>
           searchConverterService.articleHitAsMultiSummary(hit.sourceAsString, language)
-        case `draftType` =>
+        case Some(`draftType`) =>
           searchConverterService.draftHitAsMultiSummary(hit.sourceAsString, language)
-        case `learningPathType` =>
+        case Some(`learningPathType`) =>
           searchConverterService.learningpathHitAsMultiSummary(hit.sourceAsString, language)
```

Comment:
Ser at vi får compiler warning på at denne ikke er exhaustive (Skjønner egentlig ikke hvorfor vi ikke fikk det før), men kanskje vi burde endre denne til å returnere en `Option` og returnere en `None` og logge en feil på `case _`?

Evt gjøre det til en `Try` og feile hele søket da for å oppdage eventuelle feil fortere, men det er kanskje litt vel drastisk?

Reply:
Det feiler også når eg kjører denne koden mot en indeks generert på 6.8.4. Ser om eg kjem meg rundt det.

Reply:
Oppdaterte koden slik at den kan kjøre både mot 6-indeks og 7-indeks.

Reply:
Ser 65 tester feiler om jeg endrer testene til å bruke en `6.8.4` container, men det er kanskje indeksering som feiler, mens vanlige søk fungerer? Da burde i så fall clusteret oppgraderes relativt fort etter denne blir deployet? :)

Reply:
Det stemmer. Indeksering feiler på eldre elasticsearch, men søket fungerer. Cluster må oppgraderes etter deploy ja.

Reply:
:+1: